exasol_integration_test_docker_environment/lib/test_environment/create_certificates/analyze_certificate_container.py

```diff
@@ -91,7 +96,7 @@ def run_task(self):
 
 class DockerTestContainerPush(DockerCertificateBuildBase, DockerPushParameter):
 
-    def run_task(self):
+    def run_task(self) -> Iterator[DockerCreateImageTask]:
```

Comment:
`DockerCreateImageTask` is not 100% correct, as it can be different classes. Maybe in another PR...